CRITICAL - 2023-03-04 00:00:47 --> syntax error, unexpected token "public"
in APPPATH\Controllers\Guest.php on line 24.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Guest.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Guest')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Guest')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\Guest', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-04 00:00:48 --> syntax error, unexpected token "public"
in APPPATH\Controllers\Guest.php on line 24.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Guest.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Guest')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Guest')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\Guest', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-03-04 00:00:48 --> syntax error, unexpected token "public"
in APPPATH\Controllers\Guest.php on line 24.
 1 SYSTEMPATH\Autoloader\Autoloader.php(289): CodeIgniter\Autoloader\Autoloader->includeFile('APPPATH\\Controllers\\Guest.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(267): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Guest')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Guest')
 4 SYSTEMPATH\CodeIgniter.php(892): class_exists('\\App\\Controllers\\Guest', true)
 5 SYSTEMPATH\CodeIgniter.php(486): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-04 01:31:08 --> mysqli_sql_exception: Table 'webprogmi211.eplascano_guests' doesn't exist in C:\wamp64\www\lab3\ci4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\wamp64\www\lab3\ci4\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\wamp64\www\lab3\ci4\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\wamp64\www\lab3\ci4\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\wamp64\www\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\wamp64\www\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\wamp64\www\lab3\ci4\app\Models\GuestModel.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\wamp64\www\lab3\ci4\app\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
#8 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\Guest->index()
#9 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
#10 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp64\www\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-04 01:31:08 --> Table 'webprogmi211.eplascano_guests' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `eplascano_guests`', [], false)
 2 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\GuestModel.php(15): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Guest->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-04 01:50:14 --> mysqli_sql_exception: Table 'webprogmi211.eplascano_guests' doesn't exist in C:\wamp64\www\lab3\ci4\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\wamp64\www\lab3\ci4\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\wamp64\www\lab3\ci4\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\wamp64\www\lab3\ci4\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\wamp64\www\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\wamp64\www\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\wamp64\www\lab3\ci4\app\Models\GuestModel.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\wamp64\www\lab3\ci4\app\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
#8 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\Guest->index()
#9 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
#10 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp64\www\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-03-04 01:50:14 --> Table 'webprogmi211.eplascano_guests' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `eplascano_guests`', [], false)
 2 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\GuestModel.php(15): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Guest->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-03-04 02:06:47 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\wamp64\www\lab3\ci4\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\wamp64\www\lab3\ci4\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('192.168.150.213', 'webprogmi211', 'j@zzyAngle30', 'webprogmi211', 3306, '', 0)
#1 C:\wamp64\www\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp64\www\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp64\www\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\wamp64\www\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp64\www\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\wamp64\www\lab3\ci4\app\Models\GuestModel.php(15): CodeIgniter\BaseModel->findAll()
#7 C:\wamp64\www\lab3\ci4\app\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
#8 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\Guest->index()
#9 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
#10 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\wamp64\www\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\wamp64\www\lab3\ci4\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\wamp64\www\lab3\ci4\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp64\www\lab3\ci4\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp64\www\lab3\ci4\system\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\wamp64\www\lab3\ci4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp64\www\lab3\ci4\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\wamp64\www\lab3\ci4\app\Models\GuestModel.php(15): CodeIgniter\BaseModel->findAll()
#6 C:\wamp64\www\lab3\ci4\app\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
#7 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(934): App\Controllers\Guest->index()
#8 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
#9 C:\wamp64\www\lab3\ci4\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\wamp64\www\lab3\ci4\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2023-03-04 02:06:47 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1602): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `eplascano_guests`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\GuestModel.php(15): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Guest.php(14): App\Models\GuestModel->getGuest()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Guest->index()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Guest))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
